hibernate-core/src/main/java/org/hibernate/persister/entity/AbstractEntityPersister.java

```diff
@@ -4113,60 +4138,14 @@ protected Map getLoaders() {
 	//Relational based Persisters should be content with this implementation
 	protected void createLoaders() {
 		final Map loaders = getLoaders();
-		loaders.put( LockMode.NONE, createEntityLoader( LockMode.NONE ) );
 
-		UniqueEntityLoader readLoader = createEntityLoader( LockMode.READ );
-		loaders.put( LockMode.READ, readLoader );
+		noneLockLoader = createEntityLoader( LockMode.NONE );
+		loaders.put( LockMode.NONE, noneLockLoader );
```

Comment:
Why do both (field + Map entry) for NONE and READ?  Not understanding that part.  Is it so that callers can continue to get access to those Loaders via `#getLoaders`?  Just like discussed above, `#getLoaders` is intended to have a very specific use - a reference by which to add a new Loader.  If that's true there, its true here too

Just drop those 2 from

Reply:
I thought I added a comment about that on GitHub: I wasn't really sure about how safe we should be with the `loaders` map.

But sure, if we don't expect the user to use this map, we can drop those 2 from the map.